frontend/src/pages/modelServing/screens/metrics/bias/BiasConfigurationPage/BiasConfigurationModal/MetricTypeField.tsx

```diff
@@ -37,9 +39,15 @@ const MetricTypeField: React.FC<MetricTypeFieldProps> = ({ fieldId, value, onCha
           <SelectOption
             key={type}
             value={type}
-            description={METRIC_TYPE_DESCRIPTION[type as keyof typeof BiasMetricType]}
+            description={
+              asEnumMember(type, BiasMetricType) !== null
+                ? METRIC_TYPE_DESCRIPTION[asEnumMember(type, BiasMetricType)!]
+                : null
```

Comment:
We need a function `mapEnum(<enum>)` which should return all the enum literals as an array.
I think this would solve several other areas in in this PR related to iterating over enums.

Reply:
Can I have more clarification on what that would look like? i.e. what other functionality it will bring that `asEnumMember` doesn't?